docs/src/syntax-highlighting/index.mjs

```diff
@@ -1,6 +1,5 @@
 import { createHighlighter } from 'shiki';
 import rehypePrettyCode from 'rehype-pretty-code';
-import { rehypeEmptyLines } from './rehypeEmptyLines.mjs';
```

Comment:
So, we should delete this file too? We `rehypeEmptyLines.mjs` is orphan now?

Reply:
It has some purpose, but I'm not sure what - Vlad knows. I kept it in case it'll be re-added soon with a fixed implementation. Since git history keeps it here, it's not crucial to be kept around though.